addons/ccm-vsphere/ccm-vsphere.yaml

```diff
@@ -188,8 +188,10 @@ spec:
           args:
             - --cloud-provider=vsphere
             - --cloud-config=/etc/cloud/cloud-config
-{{ if .Config.CABundle }}
           env:
+            - name: ENABLE_ALPHA_DUAL_STACK
+              value: "{{ .Config.ClusterNetwork.IPFamily.IsDualStack }}"
```

Comment:
I think the function is called `IsDualstack` (lowercase `s`).

Reply:
Updated